<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boatAccount_dialog</class>
 <widget class="QDialog" name="boatAccount_dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boat account</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>540</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QListWidget" name="list_boat">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_login"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_pass">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="enable_state">
          <property name="text">
           <string>Actif</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Polaire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="polarList"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QPushButton" name="btn_accAdd">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_accChg">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_accDel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>list_boat</tabstop>
  <tabstop>edit_login</tabstop>
  <tabstop>edit_pass</tabstop>
  <tabstop>enable_state</tabstop>
  <tabstop>btn_accAdd</tabstop>
  <tabstop>btn_accChg</tabstop>
  <tabstop>btn_accDel</tabstop>
  <tabstop>btn_Ok</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btn_Ok</sender>
   <signal>clicked()</signal>
   <receiver>boatAccount_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>boatAccount_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_login</sender>
   <signal>textChanged(QString)</signal>
   <receiver>boatAccount_dialog</receiver>
   <slot>slot_accHasChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_pass</sender>
   <signal>textChanged(QString)</signal>
   <receiver>boatAccount_dialog</receiver>
   <slot>slot_accHasChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_state</sender>
   <signal>stateChanged(int)</signal>
   <receiver>boatAccount_dialog</receiver>
   <slot>slot_accHasChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_accHasChanged()</slot>
 </slots>
</ui>
